src/server/index.tsx

```diff
@@ -52,7 +52,7 @@ app.use(promBundle({ includeMethod: true, metricsPath: `${base}/metrics` }));
 // parse cookies
 app.use(cookieParser());
 // server static assets from the /public directory
-app.use(`${base}/public`, express.static(join(__dirname, 'public')));
+app.use(`/public`, express.static(join(__dirname, 'public')));
```

Comment:
this seems to go in the other direction; we need to have the client make use of the configured base...